node/core/approval-voting/src/lib.rs

```diff
@@ -1676,6 +1717,12 @@ fn check_and_import_assignment(
 	candidate_index: CandidateIndex,
 ) -> SubsystemResult<(AssignmentCheckResult, Vec<Action>)> {
 	let tick_now = state.clock.tick_now();
+	let span = state.spans.get(&assignment.block_hash).unwrap();
```

Comment:
We need to handle the case when this is None.

Reply:
Yeah, noticing this in the tests that fail as well - good thing they are there!

Reply:
Fixed.